Resources/config/monolog.xml

```diff
@@ -36,5 +36,9 @@
         <service id="monolog.formatter.logstash" class="Monolog\Formatter\LogstashFormatter" public="false">
             <argument index="0">app</argument>
         </service>
+
+        <service id="monolog.http_client" class="Symfony\Contracts\HttpClient\HttpClientInterface">
```

Comment:
it should be marked as `public="false"` so that the service can be removed

Reply:
Done. It's only for 3.4, right?

Reply:
only for 3.4 yes